README.rst

```diff
@@ -1,3 +1,7 @@
+.. image:: docs/assets/datachain_logotype.svg
```

Comment:
This does not look good on light mode. 

<img width="1048" alt="Screenshot 2024-09-07 at 07 26 32" src="https://github.com/user-attachments/assets/612a96d3-7d3d-4fd0-976d-724172e168f8">

Reply:
https://github.com/iterative/datachain/pull/408